src/XeroPHP/Models/PayrollAU/Employee/TaxDeclaration.php

```diff
@@ -1,5 +1,5 @@
 <?php
-namespace XeroPHP\Models\PayrollAU\Employee;
+    namespace XeroPHP\Models\PayrollAU\Employee;
```

Comment:
I know it's fussy, but fix the indent in this one line and I'll merge! Thanks for the work

Reply:
Done